config/initializers/omniauth.rb

```diff
@@ -12,7 +12,7 @@
            },
            name: 'CCID'
 
-  provider :developer unless Rails.env.production?
+  provider :developer unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
```

Comment:
You can also forget about the parentheses as rubocop will complain to you about them.

```
config/initializers/omniauth.rb:15:30: C: Style/ParenthesesAroundCondition: Don't use parentheses around the condition of an unless.  (https://github.com/bbatsov/ruby-style-guide#no-parens-around-condition)
  provider :developer unless (Rails.env.production? && !ENV['ENABLE_TMP_LOGINS'])
                             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```